diesel/src/expression/helper_types.rs

```diff
@@ -8,6 +8,8 @@ pub type SqlTypeOf<Expr> = <Expr as Expression>::SqlType;
 pub type AsExpr<Item, TargetExpr> = AsExprOf<Item, SqlTypeOf<TargetExpr>>;
 pub type AsExprOf<Item, Type> = <Item as AsExpression<Type>>::Expression;
 
+#[macro_export]
```

Comment:
Probably just due to module ordering. I'd just move the definitions here instead of the PG module and add another cfg attr